lib/cose/keys.ex

```diff
@@ -2,6 +2,30 @@ defmodule COSE.Keys.Symmetric do
   defstruct [:kty, :kid, :alg, :key_ops, :base_iv, :k]
 end
 
+defmodule ECDSASignature do
```

Comment:
In case we keep this module (see comment below about struct definition), the name should be `COSE.Keys. ECDSASignature`.